bedrock/products/templates/products/vpn/more/ip-address.html

```diff
@@ -0,0 +1,44 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "products/vpn/more/base-article.html" %}
+
+{% block page_title %}{{ ftl('vpn-ip-address-what-is-an-ip-address') }}{% endblock %}
+{% block page_desc %}{{ ftl('vpn-ip-address-mozilla-answers-all-of') }}{% endblock %}
+
+{% block article %}
+<p class="mzp-c-article-intro">{{ ftl('vpn-ip-address-every-time-you') }}</p>
+
+<h3>{{ ftl('vpn-ip-address-what-is-an-ip-address') }}</h3>
+<p>{{ ftl('vpn-ip-address-your-ip-address') }}</p>
+
+<figure class="c-article-figure">
+  {{ high_res_img('img/products/vpn/more/ipaddress-example.jpg', {'alt': ftl('vpn-ip-address-an-example-of-an-ip'), 'width': '512', 'height': '202'}) }}
+  <figcaption>{{ ftl('vpn-ip-address-if-you-look-up-your') }}</figcaption>
+</figure>
+
+<h3>{{ ftl('vpn-ip-address-does-it-travel') }}</h3>
+<p>{{ ftl('vpn-ip-address-no-your-ip') }}</p>
+
+<h3>{{ ftl('vpn-ip-address-does-your-ip') }}</h3>
+<p>{{ ftl('vpn-ip-address-yes-even-if') }}</p>
+
+<h3>{{ ftl('vpn-ip-address-can-more-than') }}</h3>
+<p>{{ ftl('vpn-ip-address-this-is-a') }}</p>
+
+<figure class="c-article-figure">
+  {{ high_res_img('img/products/vpn/more/ipaddress-devices.jpg', {'alt': '', 'width': '700', 'height': '172'}) }}
+</figure>
+
+<h3>{{ ftl('vpn-ip-address-can-we-run') }}</h3>
+<p>{{ ftl('vpn-ip-address-when-the-internet', sr='https://www.siliconrepublic.com/comms/ip-addresses-running-out', variety='https://variety.com/2019/digital/news/u-s-households-have-an-average-of-11-connected-devices-and-5g-should-push-that-even-higher-1203431225/') }}</p>
+<p>{{ ftl('vpn-ip-address-we-now-have') }}</p>
+
+<h3>{{ ftl('vpn-ip-address-should-you-hide') }}</h3>
+<p>{{ ftl('vpn-ip-address-you-dont-need', congress='https://blog.mozilla.org/en/mozilla/internet-policy/u-s-broadband-privacy-rules-will-fight-protect-user-privacy/', doh='https://support.mozilla.org/kb/firefox-dns-over-https', firefox=url('firefox.new')) }}</p>
```

Comment:
May or may not be important - should owned blog links contain utm params to track referrals?